mangaScrape.py

```diff
@@ -97,20 +85,19 @@ def linkData(url):
 if not os.path.exists(BASE_PATH):
     os.makedirs(BASE_PATH)
 
-imageUrl = saveImg(linkData(BASE_URL))
-#print imageUrl.parent['href']
+IMAGEURL = saveImg(linkData(BASE_URL))
```

Comment:
Can we move the processing to a dedicated main method? Will prevent use of lot of globals like CTR, NEXT_REL..